tgui/packages/tgui/interfaces/ListInput.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * @file
+ * @copyright 2020 watermelon914 (https://github.com/watermelon914)
+ * @license MIT
+ */
+
+import { clamp01 } from 'common/math';
+import { useBackend, useLocalState } from '../backend';
+import { Box, Button, Flex, Section, Input } from '../components';
+import { Window } from '../layouts';
+
+const ARROW_KEY_UP = 38;
+const ARROW_KEY_DOWN = 40;
```

Comment:
Can you add these to hotkeys.js and import them instead?
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/e1956ab2-19cb-49bd-89f1-2b6ac3afd322)